src/client/dfuse/ops/lookup.c

```diff
@@ -266,6 +266,7 @@ dfuse_cb_lookup(fuse_req_t req, struct dfuse_inode_entry *parent,
 	DFUSE_TRA_UP(ie, parent, "inode");
 
 	dfuse_ie_init(dfuse_info, ie);
+	dfuse_dcache_set_time(ie);
```

Comment:
I'd expect this to be set when the dcache is populated (or potentially accessed) but lookup is neither of these.  I'd need to work through the ticket to know if this was correct.